bpfhdr_bsd.go

```diff
@@ -0,0 +1,12 @@
+// +build darwin dragonfly freebsd netbsd
+
+package raw
+
+import "golang.org/x/sys/unix"
+
+type bpfHdr struct {
```

Comment:
There should be a `unix.BpfHdr` so we don't have to deal with this structure's definition.

Reply:
Good suggestion, done. The ”unix” package is, to say the least, not very well documented.